src/quokka_editor_back/routers/documents.py

```diff
@@ -49,50 +33,47 @@ async def get_document(document_id: UUID) -> Document:
         )
 
 
-@router.get("/")  # TODO: add response_model
-async def read_all():
-    return (
-        await Document.all()
-    )  # TODO: add pydantic model as a serializer and reduce response objects to
+@router.get("/")
+async def read_all(current_user: Annotated[User, Depends(get_current_user)]):
+    return await Document.all()
 
 
-@router.post("/")  # TODO: add response_model
+@router.post("/")
 async def create_document(
-    document_payload: DocumentSchema,
```

Comment:
Why did you delete this?

Reply:
The idea behind this was to create document like in google docs style. Without providing anything. Just click on the desired template and create a document that is named draft